### ISSUE TITLE
'HueBridge' object has no attribute 'sensor_manager'

### ISSUE THREAD

**Issue by sebbebebbe** [robmarkcole/Hue-remotes-HASS#11]
Hi,

I am fairly new to HA. Im getting this error stated below. Anything I am missing?

My configuration looks like this:
hue:
  bridges:
    - host: 192.168.x.x
      allow_unreachable: true
      allow_hue_groups: true

remote:
  - platform: hueremote

Log Details (ERROR)
Wed Apr 08 2020 09:13:21 GMT+0200 (Central European Summer Time)
Error while setting up platform hueremote
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.6/site-packages/homeassistant/helpers/entity_platform.py", line 150, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/lib/python3.6/asyncio/tasks.py", line 358, in wait_for
    return fut.result()
  File "/home/homeassistant/.homeassistant/custom_components/hueremote/remote.py", line 40, in async_setup_platform
    config.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL),
  File "/home/homeassistant/.homeassistant/custom_components/hueremote/data_manager.py", line 159, in async_add_platform_entities
    async for is_new, model, dev_id, _ in self._iter_data(platform_models):
  File "/home/homeassistant/.homeassistant/custom_components/hueremote/data_manager.py", line 56, in _iter_data
    await bridge.sensor_manager.coordinator.async_request_refresh()
AttributeError: 'HueBridge' object has no attribute 'sensor_manager'

**Comment by biver0** [comment]
Did you copy the hueremote folder from here:

https://github.com/robmarkcole/Hue-remotes-HASS/tree/master/custom_components

to your custom_components folder and then restart home assistant?

Also, I don't think you need that hue code, just put:

discovery:

in your configuration.yaml file and HA will detect your hue bridge and lights, etc. and you can set it up with the gui.  You may need hass.io to do that, so I could be a little off here.  

Once you do that, you can just add the following to your configuration.yaml:

remote:
   \-     platform: hueremote

Someone else can say this more elequently than I.

**Comment by sebbebebbe** [comment]
I used the HACS install process and the files are indeed in place i that folder.

I forgot to mention that i use the integrated hue integration with great success. That is for sensors, lights and groups.

Ill try adding the discovery: part and see if i get somewhere.

**Comment by biver0** [comment]
Did you update HA to the latest as well?  I just did this and it worked perfectly as I described.

The discovery part was what I was suggesting would find the integrated Hue stuff.  If that's already working then a reboot after HACS install of custom component and the remote: and platform in configuration.yaml is all that is needed.  Did you check in the Lovelace "unused entities" to see if found your remotes?
